src/Drivers/Db/Queue.php

```diff
@@ -249,7 +254,7 @@ private function moveExpired()
             $this->db->createCommand()->update(
                 $this->tableName,
                 ['reserved_at' => null],
-                '[[reserved_at]] < :time - [[ttr]] and [[reserved_at]] is not null and [[done_at]] is null',
+                '[[reserved_at]] < :time - [[ttr]] and [[done_at]] is null',
```

Comment:
Is there a reason for this change?

Reply:
This was a backport of https://github.com/yiisoft/yii2-queue/commit/eb8c7d4c635ae9da1c09e4fb3ae41452e33bed8b